src/autolabel/labeler.py

```diff
@@ -651,3 +734,13 @@ def get_all_annotations(self):
             return [pickle.loads(a.llm_annotation) for a in db_result]
         else:
             return self.all_annotations
+
+    def get_num_tokens(self, inp: str) -> int:
+        """Returns the number of tokens in the prompt"""
+        return len(self.confidence_tokenizer.encode(str(inp)))
+
+    def chunk_string(self, inp: str, chunk_size: int) -> List[str]:
```

Comment:
move to utils.py ?

Reply:
uses the tokenizer, for encoding and decoding so kept it here